server/api/ballot_manifest.py

```diff
@@ -164,16 +161,11 @@ def process() -> None:
 
         # If batch tallies file already uploaded, try reprocessing it, since it
         # depends on batch names from the manifest
-        if jurisdiction.batch_tallies_file:
-            clear_batch_tallies_data(jurisdiction)
-            jurisdiction.batch_tallies_file.task = create_background_task(
-                process_batch_tallies_file,
-                dict(
-                    jurisdiction_id=jurisdiction.id,
-                    jurisdiction_admin_email=jurisdiction_admin_email,
-                    support_user_email=support_user_email,
-                ),
-            )
+        reprocess_batch_tallies_file_if_uploaded(
+            jurisdiction,
+            (UserType.JURISDICTION_ADMIN, jurisdiction_admin_email),
+            support_user_email,
+        )
```

Comment:
We're already reprocessing candidate-totals-by-batch files when ballot manifests are updated. This PR just moves the reprocessing logic to a shared location and applies it when updating contests as well